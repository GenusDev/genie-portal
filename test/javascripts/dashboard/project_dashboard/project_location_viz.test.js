import React from 'react'
import ProjectGraph from 'Entities/dashboard/project_dashboard/project_graph'
import { shallow, mount, render } from 'enzyme';

function setup(){
  const props = {
    fetchProjects: jest.fn(()=>{
      return Promise.resolve();
    })
  }
  const eWrapper = mount(<ProjectGraph {...props}/>)
  return {
    props,
    eWrapper
  }
}


// describe what we are testing
describe('Project Graph Component', () => {
 // make our assertion and what we expect to happen


 it('should render without throwing an error', () => {

   const { eWrapper} = setup()
   //
   // expect(eWrapper.find('section').hasClass('graph-container')).toBe(true)

    expect(eWrapper).toMatchSnapshot();




 })
})

test('1 + 1 equals 2', () => {
  expect(1 + 3).toBe(4);
});
