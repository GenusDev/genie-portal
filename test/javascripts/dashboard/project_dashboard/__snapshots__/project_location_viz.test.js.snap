// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Project Graph Component should render without throwing an error 1`] = `
<ProjectGraph
  chart="loading"
  fetchProjects={
    [MockFunction] {
      "calls": Array [
        Array [],
      ],
      "results": Array [
        Object {
          "isThrow": false,
          "value": Promise {},
        },
      ],
    }
  }
>
  <div
    className="graph-container"
  >
    <div
      className="series content graph"
      id="project"
    >
      <div
        id="graph"
      />
    </div>
  </div>
</ProjectGraph>
`;
